backend-api/src/functions/adapters/fetchAdapter.ts

```diff
@@ -0,0 +1,7 @@
+export const fetchAdapter: IFetchAdapter = (url, options) => {
```

Comment:
Not how this implementation will work. Aiming to somewhat replicate functionality here but without exponential back off: https://github.com/govuk-one-login/sts-back/blob/main/backend-api/src/common/http/sendHttpRequest.ts

Can I check before I do this that you're happy with this @sandymay-dd re the retry strategy. If we will eventually want exponential back off here then might be easy to lift the entire func instead of refactoring to be simpler. If we do just want a linear retry strat then I will do the refactor 

cc: @ahickmann-gds

Reply:
FYI currently the retry stuff is in the token service

Reply:
Am doing a simplified version FYI.